js/pathfinding.js

```diff
@@ -178,7 +197,8 @@ function stepCollidesWithWall(from, to, token) {
 }
 
 export function wipePathfindingCache() {
-	cache.clear();
+	cache.nodes.clear();
+	cache.cacheIds.clear();
```

Comment:
This could be a single `clear` command, with the sub-clears being handled by the `cache` class